### PR TITLE
fix(docs): update link to using-gatsby-without-graphql example

### PR DESCRIPTION
**Pull request by aceyoung9**
## Description

Two of the examples link to each other, and one of them was renamed; this fixes a lingering dead-link.

## Related Issues

Related to #10904, #10782

Thanks!

### CODE REVIEW

File: examples/using-local-plugins/README.md [hunk 1]
```diff
@@ -16,4 +16,4 @@ This example uses a [local plugin](/docs/loading-plugins-from-your-local-plugins
 2. Process that data into Gatsby's node format
 3. Use the [`createNode` action](/docs/actions/#createNode) to add the data to Gatsby’s GraphQL layer
 
-The [`gatsby-node.js` file](https://github.com/jlengstorf/gatsby-with-unstructured-data/blob/using-gatsby-data-layer/plugins/gatsby-source-pokeapi/gatsby-node.js) of the local plugin includes detailed comments on the process.
+The [`gatsby-node.js` file](./plugins/gatsby-source-pokeapi/gatsby-node.js) of the local plugin includes detailed comments on the process.
```
Comment (laurieontech) [inline examples/using-local-plugins/README.md]: This is not an internal link, so should not be a relative path.

File: examples/using-local-plugins/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ You might also be interested in the docs section on [local plugins](/docs/creati
 
 ## Using Gatsby's GraphQL integration layer
 
-This example site is also intended as a direct comparison to the [using-unstructured-data example](../using-unstructured-data), which illustrates how to use an "unstructured data" approach (or, without making use of the GraphQL integration layer).
+This example is also intended as a direct comparison to the [using-gatsby-without-graphql](../using-gatsby-without-graphql) example.
```
Comment (laurieontech) [inline examples/using-local-plugins/README.md]: Happy to fix the dead link, but can we keep the text?

**Comment by laurieontech** [comment]
Thanks for the PR and for catching this. Left a couple of comments.

File: examples/using-local-plugins/README.md (hunk 1 above)
Comment (aceyoung9) [reply examples/using-local-plugins/README.md]: Wait, should it be a relative path, though? Seems like a reader would expect to be linked to the `gatsby-node.js` file of this local plugin in this example, not the file in the external repo that these examples were pulled from.

Perhaps if it's important to leave a link to jlengstorf's repo, we should just do that in a separate section?

Comment (laurieontech) [reply examples/using-local-plugins/README.md]: You're correct. I missed the context! Good catch.

File: examples/using-local-plugins/README.md (hunk 2 above)
Comment (laurieontech) [inline examples/using-local-plugins/README.md]:
```suggestion
This example is also intended as a direct comparison to the [using-gatsby-without-graphql](../using-gatsby-without-graphql) example, which illustrates how to use an "unstructured data" approach (or, without making use of the GraphQL integration layer).
```

**Comment by laurieontech** [comment]
Hey @aceyoung9, if we can accept these two changes this is ready to be merged!

**Comment by vladar** [comment]
@aceyoung9 This PR got a bit stuck for a while. Could you please check the suggestion of @laurieontech ? If you don't have time to work on this, maybe you can provide access to your fork so that we could continue with it?

**Comment by aceyoung9** [comment]
@vladar @laurieontech Hi, sorry, I will unstick this PR and rebase appropriately in the next 24 hours.

**Pull request merged**